src/libPMacc/include/nvidia/functors/Assign.hpp

```diff
@@ -22,6 +22,8 @@
 
 #pragma once
 
+#include "pmacc_types.hpp"
+
 namespace PMacc
 {
     namespace nvidia
```

Comment:
yes, but this PR is, as stated in the description, a bug fix and tries to fix exactly *one* thing.

all files inside `src/libPMacc/include/nvidia/functors` have wrong formatting, we are not picking things we come along in our PRs but we change focused and topic-based.